submissions/aymkin/OOP-frogger/app.js

```diff
@@ -0,0 +1,143 @@
+let scoreCount = 0;
+
+const playerStartValues = {
+	x: 202,
+	y: 405,
+	sprite: 'images/char-boy.png',
+};
+
+const enemyStartValues = {
+	x: 0,
+	y: [63, 146, 229],
+	sprite: 'images/enemy-bug.png',
+};
+
+const FIELD = {
+	column: 101,
+	row: 83,
+	leftBorder: 101,
+	rightBorder: 303,
+	topBorder: 0,
+	bottomBorder: 322,
+};
+
+const Creature = function(x, y, sprite) {
+	this.x = x;
+	this.y = y;
+	this.sprite = sprite;
+};
+
+Creature.prototype.render = function() {
+	ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+const Enemy = function(x, y, sprite, gamer) {
+	Creature.call(this, x, y, sprite);
+	this.speed = this.getRandSpeed();
+	this.gamer = gamer;
+};
+
+Enemy.prototype = Object.create(Creature.prototype);
+
+Enemy.prototype.getRandSpeed = function(min = 75, max = 200) {
+	return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+Enemy.prototype.update = function(dt) {
+	this.x = this.x + this.speed * dt;
```

Comment:
As for me, it is more readable syntax like `this.x += this.speed * dt`, but it is up to you.